com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/pyspltuple.pm

```diff
@@ -0,0 +1,243 @@
+use Switch;
+
+# This function takes the string of a value to be converted and its
+# type, and generates the corresponding code to convert the type from
+# c++ to Python. Note that the $type argument is a SPL::CodeGen type,
+# and not a string literal
+#
+sub cppToPythonPrimitiveConversion{
```

Comment:
These seems to be an almost copy of the existing code, is there a way we could refactor to have a single version?